server/src/controllers/pollController.ts

```diff
@@ -108,17 +109,15 @@ async function getStudents(courseCode: string, startTime: Date, endTime: Date) {
   }
 }
 
-async function getPollStatus(pollId: any) {
-  if (pollId === null || pollId === undefined || typeof pollId !== "string")
-    return { status: 400, data: { message: "Invalid poll Id" } };
+async function getPollStatus(pollId: string) {
+  if (!pollId) return { status: 400, data: { message: "Invalid poll Id" } };
   const result = await client.get(pollId);
   const pollStarted = result === null ? false : parseInt(result) > 0;
   return { status: 200, data: { pollStarted } };
 }
 
-async function getResult(pollId: any) {
-  if (pollId === null || pollId === undefined || typeof pollId !== "string")
-    return { status: 400, data: { message: "Invalid poll Id" } };
+async function getResult(pollId: string) {
+  if (!pollId) return { status: 400, data: { message: "Invalid poll Id" } };
```

Comment:
If `pollId` is never undefined then we don't really need this if statement right?

Reply:
Yes, good catch!

Reply:
What if `pollId === ''`? Shouldn't this catch it?

Reply:
Thats true but in that case i would make this comparison explicitly just because it makes the code more readable